### PR TITLE
* Add 9_1_0_39497 support

### PR DESCRIPTION
**Pull request by Mistixx**

### CODE REVIEW

File: WowPacketParser/Enums/ClientVersionBuild.cs [hunk 1]
```diff
@@ -349,6 +349,7 @@ public enum ClientVersionBuild
         V9_1_0_39318 = 39318,
         V9_1_0_39335 = 39335,
         V9_1_0_39427 = 39427,
+	    V9_1_0_39497 = 39497,
```
Comment (Faq) [inline WowPacketParser/Enums/ClientVersionBuild.cs:352]:
```suggestion
        V9_1_0_39427 = 39427,
        V9_1_0_39497 = 39497,
```

**Comment by Shauren** [comment]
Replace tabs with spaces

File: WowPacketParser/Enums/Version/Opcodes.cs [hunk 2]
```diff
@@ -483,6 +483,7 @@ public static ClientVersionBuild GetOpcodeDefiningBuild(ClientVersionBuild build
                 case ClientVersionBuild.V9_1_0_39318:
                 case ClientVersionBuild.V9_1_0_39335:
                 case ClientVersionBuild.V9_1_0_39427:
+				case ClientVersionBuild.V9_1_0_39497:
```
Comment (funjoker) [inline WowPacketParser/Enums/Version/Opcodes.cs:486]: format

File: WowPacketParser/Misc/ClientVersion.cs [hunk 3]
```diff
@@ -364,6 +364,7 @@ public static class ClientVersion
             new KeyValuePair<ClientVersionBuild, DateTime>(ClientVersionBuild.V9_1_0_39318, new DateTime(2021, 07, 06, 23, 40, 00)),
             new KeyValuePair<ClientVersionBuild, DateTime>(ClientVersionBuild.V9_1_0_39335, new DateTime(2021, 07, 08, 19, 22, 00)),
             new KeyValuePair<ClientVersionBuild, DateTime>(ClientVersionBuild.V9_1_0_39427, new DateTime(2021, 07, 16, 20, 06, 00)),
+			new KeyValuePair<ClientVersionBuild, DateTime>(ClientVersionBuild.V9_1_0_39497, new DateTime(2021, 07, 26, 03, 06, 00)),
```
Comment (funjoker) [inline WowPacketParser/Misc/ClientVersion.cs:367]: format

File: WowPacketParser/Enums/Version/Opcodes.cs [hunk 4]
```diff
@@ -483,6 +483,7 @@ public static ClientVersionBuild GetOpcodeDefiningBuild(ClientVersionBuild build
                 case ClientVersionBuild.V9_1_0_39318:
                 case ClientVersionBuild.V9_1_0_39335:
                 case ClientVersionBuild.V9_1_0_39427:
+			    case ClientVersionBuild.V9_1_0_39497:
```
Comment (funjoker) [inline WowPacketParser/Enums/Version/Opcodes.cs:486]: format

**Pull request closed without merging**